sir_complainsalot/delphi_sir_complainsalot/run.py

```diff
@@ -37,14 +37,6 @@ def run_module():
                                        params["sources"], params.get("grace", 0), LOGGER))
 
     if len(complaints) > 0:
-        for complaint in complaints:
-            LOGGER.critical(event="signal out of SLA",
```

Comment:
Why does this need to be moved?

Reply:
Old: each line-level complaint was being logged one at a time (one line per type/source/signal/geo combo)
New: only the aggregated complaint is logged (one line per type/source; signal/geo aggregated)

(this makes it much more useful for alerting)